src/main/java/org/jenkinsci/plugins/nomad/NomadCloud.java

```diff
@@ -177,8 +177,8 @@ private void pruneOrphanedWorkers(NomadWorkerTemplate template) {
                     String jobNamespace = worker.getJobSummary().getNamespace();
                     JSONObject job = this.nomad.getRunningWorker(jobSummary.getJobID(), jobNamespace);
                     String jobRegion = job.getString("Region");
-                    Instant expiryTime = Instant.ofEpochMilli(job.getLong("SubmitTime"));
-                    expiryTime.plusSeconds(this.workerTimeout * 60);
+                    Instant expiryTime = Instant.ofEpochMilli(Math.round(job.getLong("SubmitTime")/1000000));
```

Comment:
Thanks for the fix! You have to fix the "SpotBugs-finding" first in order to get the build the green.

Suggestion, just remove Math.round like below:
```
Instant expiryTime = Instant.ofEpochMilli(job.getLong("SubmitTime") / 1000000);
expiryTime = expiryTime.plusSeconds(this.workerTimeout * 60L);
```

Reply:
I used math.Float because ofEpochMilli is expececting a long and the divide to convert to ns generate a number with decimals result of  a BigDecimal type. The Math.Round solved both issues (remove the decimals + return a long).
I pushed a different way to do it, hope it's a better way ! :D 

```
groovy.lang.MissingMethodException: No signature of method: static java.time.Instant.toEpochMilli() is applicable for argument types: (java.math.BigDecimal) values: [1671487054166.230409]
Possible solutions: toEpochMilli(), ofEpochMilli(long)
```

Reply:
I adjusted the expiry time calculation so that it is more clear. Could you please check whether this is working for you?

Reply:
It's working :D !